security-oauth2/src/main/java/io/micronaut/security/oauth2/client/ClientUtils.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.security.oauth2.client;
+
+import io.micronaut.core.convert.value.ConvertibleMultiValues;
+import io.micronaut.core.convert.value.MutableConvertibleMultiValuesMap;
+import io.micronaut.http.HttpRequest;
+
+import java.util.Map;
+
+/**
+ * Utility methods for clients.
+ *
+ * @author Álvaro Sánchez-Mariscal
+ */
+final class ClientUtils {
```

Comment:
Is this package-private by design? or does it miss the `public` modifier?

Reply:
Yes. No need to be public